client/public/static/css/login-modal.css

```diff
@@ -64,3 +63,21 @@ a,
 a:hover {
   color: #333;
 }
+
+.form-control-v1{
+  width: 100%;
+  padding: 6px 12px;
+  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(13, 111, 224, 0.6);
+}
+
+.input-group input {
+  padding: 6px 12px;
+}
+
+.input-group-addon {
+  position: absolute;
+  top: 50%;
+  right: 10px;
+  transform: translateY(-50%);
+  border: none;
+}
```

Comment:
Try to extend the `form-control` class instead of creating a new one because the form trickles down throughout. Ideally, we want to keep the existing view of the forms and input group BUT add the eye icon into only the password text fields.

Other that that good work on the first issue!

Old:
<img width="498" alt="Screen Shot 2024-07-06 at 7 41 29 PM" src="https://github.com/What-Should-I-Eat/WSIE/assets/18201159/30bfd5e1-a6a6-4ca5-86e1-44be9d80b6ff">

New:
<img width="499" alt="Screen Shot 2024-07-06 at 7 41 47 PM" src="https://github.com/What-Should-I-Eat/WSIE/assets/18201159/31001de2-af4f-43d0-977b-5eb52d4bbf3d">

Reply:
Removed dependency on new class. I ended up modifying HTML layout instead to fix the issue.